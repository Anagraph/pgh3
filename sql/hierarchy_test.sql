create extension if not exists postgis;
create extension if not exists pgh3;


/* hierarchy */

select h3_to_parent('85639c63fffffff', 4);

select h3_to_parent('85639c63fffffff', 1);

select h3_to_children('82639ffffffffff', 3);
