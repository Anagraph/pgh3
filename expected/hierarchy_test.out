create extension if not exists postgis;
NOTICE:  extension "postgis" already exists, skipping
create extension if not exists pgh3;
NOTICE:  extension "pgh3" already exists, skipping
/* hierarchy */
select h3_to_parent('85639c63fffffff', 4);
  h3_to_parent   
-----------------
 84639c7ffffffff
(1 row)

select h3_to_parent('85639c63fffffff', 1);
  h3_to_parent   
-----------------
 8163bffffffffff
(1 row)

select h3_to_children('82639ffffffffff', 3);
 h3_to_children  
-----------------
 836398fffffffff
 836399fffffffff
 83639afffffffff
 83639bfffffffff
 83639cfffffffff
 83639dfffffffff
 83639efffffffff
(7 rows)

